theologie-und-philosophie.csl

```diff
@@ -238,61 +272,64 @@
       <key macro="date"/>
     </sort>
     <layout suffix=".">
-      <text macro="author" suffix=", " font-variant="small-caps"/>
-      <choose>
-        <if type="entry-encyclopedia">
-          <text variable="title" prefix="Art. " quotes="true" font-style="italic"/>
-        </if>
-        <else>
-          <text variable="title" font-style="italic"/>
-        </else>
-      </choose>
+      <text macro="author" suffix=", "/>
+      <text macro="title" font-style="italic"/>
       <choose>
         <if type="webpage">
-          <text variable="container-title" prefix=", online unter: "/>
+          <text variable="container-title" prefix=". Online unter: "/>
           <text variable="URL" prefix=", "/>
           <group delimiter="; " prefix=" (" suffix=")">
             <text macro="date" prefix="Stand: "/>
             <text macro="accessed"/>
           </group>
         </if>
         <else-if type="speech" match="any">
-          <text value=", "/>
+          <text value=". "/>
```

Comment:
could we find a better solution for this? I really don't like punctuation in text values. How about putting a group on the three rows below and adding a `prefix=". "`?